web/public/locales/en.json

```
@@ -1009,6 +1011,23 @@
+    "searchUsers": "Search users...",
+    "sessions": "Sessions",
+    "xSessions": "{{x}} sessions",
+    "avgDuration": "Avg. duration",
+    "firstSeen": "First seen",
+    "lastSeen": "Last seen",
+    "activityCalendar": "Activity calendar",
+    "topPages": "Top pages",
+    "locationAndDevice": "Location & Device",
+    "userSessions": "Sessions",
+    "noSessions": "No sessions found for this user.",
+    "noData": "No data available",
```

Biome (2.1.2)

[error] The key noData was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)